packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -1386,6 +1388,53 @@ describe('TransactionController', () => {
         expect(status).toBe(TransactionStatus.rejected);
       });
     });
+
+    describe('checks from address origin', () => {
+      it('throws if `from` address is different from current selected address', async () => {
+        const controller = newController();
+        const origin = ORIGIN_METAMASK;
+        const fromMocked = ACCOUNT_2_MOCK;
```

Comment:
Nit: We could've rename it to `notSelectedFromAddress` to make it explicit so it's easier to read test imo